src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -467,6 +468,11 @@ export default {
 		EventBus.$on('should-refresh-conversations', this.handleShouldRefreshConversations)
 		EventBus.$once('conversations-received', this.handleConversationsReceived)
 		EventBus.$on('route-change', this.onRouteChange)
+		// Check filter status in previous sessions
+		if (BrowserStorage.getItem('filterEnabled')) {
+			const filter = BrowserStorage.getItem('filterEnabled')
+			this.handleFilter(filter)
+		}
```

Comment:
```suggestion
			this.handleFilter(BrowserStorage.getItem('filterEnabled'))
```

If item is not in the storage, it returns null otherwise, and we acan handle null in the method